main.go

```diff
@@ -10,7 +10,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/kotanetes/go-test-it/model"
+	model "github.com/kotanetes/go-test-it/model2"
```

Comment:
@kotanetes : why it has package name as mode12

Reply:
I renamed package as model2

Reply:
is there any specific reason?  Apart from this everything looks good